code/modules/mob/abstract/ghost/storyteller/storyteller_verbs.dm

```diff
@@ -0,0 +1,346 @@
+		if(mob_name)
+			M = client_mobs[mob_name]
+
+	if(!M)
+		return
+
+	var/msg = html_decode(sanitize(tgui_input_text(src, "What do you want to narrate?", "Direct Narrate", max_length = MAX_PAPER_MESSAGE_LEN)))
+
+	if(!msg)
+		return
+
+	to_chat(M, msg)
+	log_admin("DirectNarrate: [key_name(usr)] to ([M.name]/[M.key]): [msg]")
+	message_admins(SPAN_NOTICE(SPAN_BOLD("DirectNarrate: [key_name(usr)] to ([M.name]/[M.key]): [msg]")), 1)
+
+/mob/abstract/ghost/storyteller/verb/toggle_build_mode()
+	set name = "Toggle Build Mode"
+	set category = "Storyteller"
+
+	if(usr != src)
+		return
+
+	var/datum/click_handler/handler = GetClickHandler()
+	if(handler.type == /datum/click_handler/build_mode)
+		usr.PopClickHandler()
+	else
+		usr.PushClickHandler(/datum/click_handler/build_mode)
+
+/mob/abstract/ghost/storyteller/verb/command_report()
+	set name = "Create Command Report"
+	set category = "Storyteller"
+
+	var/reporttitle = sanitizeSafe(tgui_input_text(usr, "Pick a title for the report.", "Title"))
+	if(!reporttitle)
+		reporttitle = "NanoTrasen Update"
+	var/reportbody = sanitize(tgui_input_text(usr, "Please enter anything you want. Anything. Serious.", "Body", multiline = TRUE), extra = FALSE)
+	if(!reportbody)
+		return
+
+	var/announce = tgui_alert(usr, "Should this be announced to the general population?", "Announcement", list("Yes","No"))
+	switch(announce)
+		if("Yes")
+			command_announcement.Announce("[reportbody]", reporttitle, new_sound = 'sound/AI/commandreport.ogg', msg_sanitized = 1);
+		if("No")
+			to_world(SPAN_WARNING("New [SSatlas.current_map.company_name] Update available at all communication consoles."))
+			sound_to(world, ('sound/AI/commandreport.ogg'))
+
+	log_admin("Storyteller [key_name(src)] has created a command report: [reportbody]")
+	message_admins("Storyteller [key_name_admin(src)] has created a command report", 1)
+
+	//New message handling
+	post_comm_message(reporttitle, reportbody)
+
+/mob/abstract/ghost/storyteller/verb/send_odyssey_message()
+	set name = "Send Odyssey Message"
+	set category = "Storyteller"
+
+	var/reporttitle = sanitizeSafe(tgui_input_text(usr, "Pick a title for the message the Horizon will get.", "Title"))
+	if(!reporttitle)
+		reporttitle = "SCC Sensors Report"
+	var/reportbody = sanitize(tgui_input_text(usr, "Enter the message the Horizon will get. Note that they will automatically receive the sensors data for the overmap object if applicable.", "Body", multiline = TRUE), extra = FALSE)
+	if(!reportbody)
+		return
+
+	var/announce = tgui_alert(usr, "Should this be announced to the general population?", "Announcement", list("Yes","No"))
+	switch(announce)
+		if("Yes")
+			command_announcement.Announce("[reportbody]", reporttitle, new_sound = 'sound/AI/commandreport.ogg', msg_sanitized = 1);
+		if("No")
+			to_world(SPAN_WARNING("New [SSatlas.current_map.company_name] Update available at all communication consoles."))
+			sound_to(world, ('sound/AI/commandreport.ogg'))
```

Comment:
`sound_to_playing_players` ?
Presumably you don't want the people in the lobby to hear it